### DISCUSSION TITLE
Chirpity funktioniert nicht mit Lenovo M70q i5-10505

### DISCUSSION THREAD

**Discussion by manud7889** [Mattk70/Chirpity-Electron#174] [Q&A]
Hallo zusammen,
ich bekomme Chirpity auf meinem Mini PC nicht zum Laufen. Es gibt schon Probleme beim Ordner Öffnen, es passiert nix. Wenn ich die Dateien einlesen kann (nach hunderten Versuchen), funktioniert die Analyse nicht :-(
Auch kommt zu Beginn die eine Fehlermeldung bezüglich einer Experimentellen webGPU?
Könnt ihr mir bitte helfen !?

**Comment by Vollpflock** [comment]
Kann deinen Anhang nicht öffnen.

**Comment by manud7889** [comment]
mein System.

Betriebsystemname	Microsoft Windows 11 Pro
Version	10.0.26100 Build 26100
Weitere Betriebsystembeschreibung 	Nicht verfügbar
Betriebsystemhersteller	Microsoft Corporation
Systemname	MANUEL
Systemhersteller	LENOVO
Systemmodell	11DUS6F00Y
Systemtyp	x64-basierter PC
System-SKU	LENOVO_MT_11DU_BU_Think_FM_ThinkCentre M70q
Prozessor	Intel(R) Core(TM) i5-10505 CPU @ 3.20GHz, 3192 MHz, 6 Kern(e), 12 logische(r) Prozessor(en)
BIOS-Version/-Datum	LENOVO M2WKT5FA, 16.07.2024
SMBIOS-Version	3.2
Version des eingebetteten Controllers	1.24
BIOS-Modus	UEFI
BaseBoard-Hersteller	LENOVO
BaseBoard-Produkt	316E
BaseBoard-Version	NOK
Plattformrolle	Desktop
Sicherer Startzustand	Ein
PCR7-Konfiguration	Erweiterung zum Anzeigen erforderlich
Windows-Verzeichnis	C:\WINDOWS
Systemverzeichnis	C:\WINDOWS\system32
Startgerät	\Device\HarddiskVolume1
Gebietsschema	Deutschland
Hardwareabstraktionsebene	Version = "10.0.26100.1"
Benutzername	MANUEL\User
Zeitzone	Mitteleuropäische Zeit
Installierter physischer Speicher (RAM)	16,0 GB
Gesamter physischer Speicher	15,7 GB
Verfügbarer physischer Speicher	11,4 GB
Gesamter virtueller Speicher	18,6 GB
Verfügbarer virtueller Speicher	14,4 GB
Größe der Auslagerungsdatei	2,88 GB
Auslagerungsdatei	C:\pagefile.sys
Kernel-DMA-Schutz	Ein
Virtualisierungsbasierte Sicherheit	Wird ausgeführt...
Virtualisierungsbasierte Sicherheit – erforderliche Sicherheitseigenschaften	
Virtualisierungsbasierte Sicherheit – verfügbare Sicherheitseigenschaften	Allgemeine Virtualisierungsunterstützung, Sicherer Start, DMA-Schutz, UEFI-Code Readonly, SMM Security Mitigations 1.0, Modusbasierte Ausführungssteuerung, APIC-Virtualisierung
Virtualisierungsbasierte Sicherheit – konfigurierte Dienste	Durch Hypervisor erzwungene Codeintegrität
Virtualisierungsbasierte Sicherheit – ausgeführte Dienste	Durch Hypervisor erzwungene Codeintegrität
App Control for Business-Richtlinie	Erzwungen
App Control for Business-Benutzermodusrichtlinie	Überwachung
Unterstützung für automatische Geräteverschlüsselung	Erweiterung zum Anzeigen erforderlich
Es wurde ein Hypervisor erkannt. Features, die für Hyper-V erforderlich sind, werden nicht angezeigt.

**Reply by Mattk70** [reply]
Having translated the above, I now suspect this has something to do with the issue:

Virtualisation-based security is running...
Virtualisation-based security – required security properties
Virtualisation-based security – available security properties: General virtualisation support, Secure Boot, DMA protection, UEFI code read-only, SMM Security Mitigations 1.0, Mode-based execution control, APIC virtualisation
Virtualisation-based security – configured services: Code integrity enforced by hypervisor
Virtualisation-based security – running services: Code integrity enforced by hypervisor
App Control for Business policy: Enforced
App Control for Business user mode policy: Monitoring
Support for automatic device encryption: Extension required to view
A hypervisor has been detected. Features required for Hyper-V are not displayed.


I'm not sure there's much I can do if your there are security policies preventing App installation or blocking certain features. The application needs write access to the filesystem. Even when using the portable version, it saves files to a config directory. Chirpity doesn't work inside the virtual machines created by Parallels, so may not work in a VM context at all.

**Comment by Mattk70** [comment]
Hi @manud7889,

For some reason, your Chirpity database is corrupt. You will need to delete the files and have Chirpity recreate them. The database is located in %APPDATA%, to find it, press the Windows key along with R, to bring up the run dialog box. In the box, type %APPDATA% and hit return, a folder will open in Windows explorer. Find the 'chirpity' folder and delete it. Then run Chirpity and things should be fine.

If you are still having issues and see the message about the experimental WebGPU backend, it is because AVX2 is not enabled on your computer. The CPU supports it, so it is possibly disabled in your BIOS. If you wish, you could try to enable AVX using the BCDEDIT command in a terminal:

This command is 
`bcdedit /set xsavedisable 0`
to turn  on AVX support 
and 
`bcdedit /set xsavedisable 1` 
to turn it off. 

To run these commands press the Windows key + X and select cmd  (open as administrator) and type them there. You will need to restart your PC afterwards for the command to take effect. **N.B.** If AVX was disabled on your computer, there may be a reason for this and it could lead to system instability - so you run this command at your own risk. 

---
Machine-translated:
Hallo @manud7889,

Aus irgendeinem Grund ist deine Chirpity-Datenbank beschädigt. Du musst die Dateien löschen, damit Chirpity sie neu erstellen kann. Die Datenbank befindet sich im Ordner %APPDATA%. Um diesen zu finden, drücke die Windows-Taste zusammen mit R, um das Ausführen-Dialogfeld zu öffnen. Gib dort %APPDATA% ein und drücke Enter. Ein Ordner wird im Windows-Explorer geöffnet. Suche den Ordner „chirpity“ und lösche ihn. Starte dann Chirpity, und alles sollte wieder funktionieren.

Wenn weiterhin Probleme auftreten und die Fehlermeldung über das experimentelle WebGPU-Backend angezeigt wird, liegt das daran, dass AVX2 auf deinem Computer nicht aktiviert ist. Die CPU unterstützt AVX2, aber möglicherweise ist es im BIOS deaktiviert. Wenn du möchtest, kannst du versuchen, AVX mit dem BCDEDIT-Befehl in einem Terminal zu aktivieren:

Der Befehl lautet:
`bcdedit /set xsavedisable 0`
um AVX zu aktivieren,
und
`bcdedit /set xsavedisable 1`
um es zu deaktivieren.

Um diese Befehle auszuführen, drücke die Windows-Taste + X und wähle „cmd“ (als Administrator öffnen). Gib die Befehle dort ein. Danach musst du deinen PC neu starten, damit die Änderungen wirksam werden. **Hinweis**: Wenn AVX auf deinem Computer deaktiviert war, könnte es dafür einen Grund geben, und die Aktivierung könnte zu Systeminstabilitäten führen – du führst diesen Befehl also auf eigenes Risiko aus.

**Comment by manud7889** [comment]
Good evening and Merry Christmas! 
Thank you for the quick tips, I did both and it helped. Now everything is working perfectly again! 

Thank you very much! 
Best wishes Manuel

**Comment by manud7889** [comment]
Danke!
